source/src/test/java/com/trustpilot/connector/dynamodb/DynamoDBSourceTaskTests.java

```diff
@@ -875,4 +877,178 @@ public void onCommitIgnoreRecordsWithoutSequenceNumber() throws InterruptedExcep
         assertEquals("", shardRegister.get("shard1").getLastCommittedRecordSeqNo());
     }
 
+    @Test
+    public void sourceInfoOfSkippedInitSyncIsLoadedFromOffsetOnStart() throws InterruptedException {
+        configs.put("init.sync.skip", "true");
+        // Arrange
+        HashMap<String, Object> offset = new HashMap<>();
+        offset.put("table_name", tableName);
+        offset.put("init_sync_state", "SKIPPED");
+        offset.put("init_sync_start", Instant.parse("2001-01-02T00:00:00.00Z").toEpochMilli());
```

Comment:
does this require putting a sync_start time even if the init sync is skipped?

Reply:
That function does require it- which shouldn't be a problem because `init sync skip` logic puts that field to 0. If it's coming from an offset, it means it will have that field already.

I'll change the test to be epoch 0.